akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -30,6 +30,10 @@ final class Sink[-In, +Mat](private[stream] override val module: Module)
 
   override val shape: SinkShape[In] = module.shape.asInstanceOf[SinkShape[In]]
 
+  def contramap[In2](f: In2 ⇒ In): Sink[In2, Mat] = Flow.fromFunction(f).toMat(this) { (_, m) ⇒ m }
+
+  def prepend[In2, Mat2](flow: Flow[In2, In, Mat2]): Sink[In2, Mat2] = flow.to(this)
```

Comment:
Please add docs as well, and a reference that this is equivalent to `Flow.to(thisSink)` if someone prefers the strict left-to-right reading.